docs/alerts.rst

```diff
@@ -0,0 +1,242 @@
+Alerts
+======
+
+Use alerts to get notifications about important App Control-monitored activities, such as the
+appearance or spread of risky files on your endpoints. The Carbon Black Cloud Python SDK provides
+all of the functionalities you might need to use them efficiently.
+You can use all of the operations shown in the API such as retrieving, filtering, dismissing, creating and updating.
+The full list of operations and attributes can be found in the :py:mod:`BaseAlert() <cbc_sdk.platform.alerts.BaseAlert>` class.
+
+For more information see
+`the developer documentation <https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/alerts-api/>`_
+
+Retrieving of Alerts
+--------------------
+
+With the example below, you can retrieve alerts with the minimum severity of ``7``.
+
+.. code-block:: python
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> from cbc_sdk.platform import BaseAlert
+    >>> api = CBCloudAPI(profile='sample')
+    >>> alerts = api.select(BaseAlert).set_minimum_severity(7)[:5]
```

Comment:
I would mention that this limits the result to the first 5 as the target for these guides are new users which may not have significant python or programming experience

Reply:
Provide a similar phrase as used in the Watchlist alert example